C:\Users\Ji\AppData\Local\Programs\Python\Python39\python.exe "C:/Program Files/JetBrains/PyCharm 2022.2.2/plugins/python/helpers/pycharm/_jb_unittest_runner.py" --path C:\Users\Ji\PycharmProjects\502pythonProject\TCSS502_Dungeon-Game\tst\adventurer-test.py 
Testing started at 8:35 AM ...
Launching unittests with arguments python -m unittest C:\Users\Ji\PycharmProjects\502pythonProject\TCSS502_Dungeon-Game\tst\adventurer-test.py in C:\Users\Ji\PycharmProjects\502pythonProject\TCSS502_Dungeon-Game\tst



Ran 9 tests in 0.008s

OK

Process finished with exit code 0
